client/ayon_core/hosts/aftereffects/api/workfile_template_builder.py

```diff
@@ -51,6 +47,8 @@ def import_template(self, path):
 class AEPlaceholderPlugin(PlaceholderPlugin):
     """Contains generic methods for all PlaceholderPlugins."""
 
+    item_type = PlaceholderItem
```

Comment:
Please use method instead of attribute.

```python
@abstractmethod
def _create_placeholder_item(self, item_data):
    pass
```

And then implement it in both plugins...
```python
def _create_placeholder_item(self, item_data):
    return CreatePlaceholderItem(
        scene_identifier=item["uuid"],
        data=item["data"],
        plugin=self
    )
```